core/src/test/java/org/opensearch/sql/expression/datetime/DateDiffTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.expression.datetime;
+
+import static java.time.temporal.ChronoUnit.DAYS;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.time.temporal.Temporal;
+import java.util.TimeZone;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class DateDiffTest extends DateTimeTestBase {
+
+  // Function signature is:
+  // (DATE/DATETIME/TIMESTAMP/TIME, DATE/DATETIME/TIMESTAMP/TIME) -> LONG
+  private static Stream<Arguments> getTestData() {
+    return Stream.of(
+        Arguments.of(LocalTime.of(12, 42), LocalTime.of(7, 40), 0L),
```

Comment:
we could pull out a lot of duplicate variables, like: 
```
// random dates for test: 
LocalTime date1 = LocalTime.of(12, 42);
LocalTime date2 = LocalTime.of(7, 40);
...

// constant dates: 
LocalTime epochStart = LocalDate.of(1970, 1, 1);
LocalTime now = LocalTime.now();
```

Reply:
Fixed in 8cd8f783, thanks.